docs/how-to/create-new-event.rst

```diff
@@ -0,0 +1,167 @@
+How to create a new Open edX Event
+==================================
+
+The mechanisms implemented by the Open edX Events library are supported and maintained by the Open edX community.
+Therefore, we've put together a guide on how to add a new event to have an effective contribution process.
+
+
+1. Propose the new event to the community
+-----------------------------------------
+
+When creating a new event, you must justify its implementation. For example, you could create a post in Discuss,
+send a message through slack or open a new issue in the library repository listing your use cases for it. Or even,
+if you have time, you could accompany your proposal with the implementation of the event to illustrate its behavior.
+
+2. Place your event in an architecture subdomain
+-------------------------------------------------
+
+As specified in the Architectural Decisions Record (ADR) events naming and versioning, the event definition needs an Open edX Architecture
+Subdomain for:
+
+- The name of the event: ``{Reverse DNS}.{Architecture Subdomain}.{Subject}.{Action}.{Major Version}``
+- The package name where the definition will live, eg. ``learning/`` or ``content_authoring/``.
+
+For those reasons, after studying your new event purpose, you must place it in one of the subdomains supported by the project:
+
++----------------+----------------------------------------------------------------------------------------------------+
+| Subdomain name | Description                                                                                        |
++================+====================================================================================================+
+| Authoring      | Allows educators to create, modify, discover, package, annotate (tag), and share learning content. |
++----------------+----------------------------------------------------------------------------------------------------+
+| Learning       | Allows learners to consume content and perform actions in a learning activity on the platform.     |
++----------------+----------------------------------------------------------------------------------------------------+
+| Discovery      | Allows learners to find the right content at the right time to help achieve their learning goals.  |
++----------------+----------------------------------------------------------------------------------------------------+
+| Programs       | Allows educators and learners to manage and engage in bundled packages (programs) of learning.     |
++----------------+----------------------------------------------------------------------------------------------------+
```

Comment:
According to the confluence page, there's another domain: Enterprise. But I don't know much about it & the page is still TBD in its definitions. I'll look it up later to fill this up